news/_posts/translation/2021-06-03-do-you-speak-prestashop-may-2021.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: post
+title: "Do you speak PrestaShop? – May 2021 edition"
+subtitle: "Where you'll learn that 1.7.8 strings are available for translation!"
+date: 2021-06-03 17:00:00
+authors: [JulieVarisellaz]
+icon: icon-world
+image: /assets/images/2019/01/Build-Crowdin-banner.jpg
+tags: [translation, i18n, l10n]
+---
+
+Contributing to PrestaShop is not only about the code, it is also about taking part in the [PrestaShop translation project](https://crowdin.com/project/prestashop-official)! This report will tell you how the translations of the software evolved in May!
+
+![Crowdin Monthly banner](/assets/images/2019/01/Build-Crowdin-banner.jpg)
+
+## Project news
+
+### New 1.7.8 strings are available for translation
+
+The catalog of 1.7.8 translations has recently been added to Crowdin. This means that the completion rate for all projects has slightly decreased. Some projects that used to be 100% translated and validated are no longer complete.
+Translators, proofreaders… your project needs you! Feel free to take a look at the new translations and help your language reach 100%!
+
+### The reward program is back
+
+In 2020, because of the sanitary crisis, our headquarters were closed and we were not able to send gifts to the top translators. But after a year, it was time to bring this reward program back to life! We also took the opportunity to reset the counters. From now on, we will only take into account the words translated from January 1, 2020. Last week, we sent about 20 packages to the top translators of the year 2020. Don’t forget to check your mailbox if this is your case. :mailbox:
+
+[Jiří VALT](https://crowdin.com/profile/gvg) already received his package, and sent us a picture of the gift with a nice message! Thank you again for your involvement in the Czech project Jirka!
+
+![Jirka's goodies reward program](/assets/images/2021/06/build-crowdin-goodies-jirka-may21.png)
```

Comment:
The uploaded file has name `assets/images/2021/06/build-crowdin-goodies-jirka-may21.png.JPG` (there is an extra `JPG` at the end). So the image does not work.